web-ui/src/main/resources/catalog/views/default/directives/directive.js

```diff
@@ -336,6 +336,12 @@
               scope.setRange();
             }
           });
+
+          scope.$on("resetSelection", function () {
```

Comment:
It works but maybe make more sense to use 
```suggestion
          scope.$on("beforeSearchReset", function () {
```
? as this is not really related to the selection. `beforeSearchReset` is triggered when search is reset.